### ISSUE TITLE
Force jg/lane nexus blitz

### ISSUE THREAD

**Issue by megastorm7** [KebsCS/KBotExt#195]
Nothing happens when I press button. Should we expect anything ?

**Comment by KebsCS** [comment]
Yes, it chooses ignite/smite for you which is disabled by default if you aren't assigned to that role
